pkg/steps/source.go

```diff
@@ -85,17 +86,21 @@ func buildFromSource(jobSpec *JobSpec, fromTag, toTag api.PipelineImageStreamTag
 					DockerStrategy: &buildapi.DockerBuildStrategy{
 						From: &coreapi.ObjectReference{
 							Kind:      "ImageStreamTag",
-							Namespace: jobSpec.Identifier(),
+							Namespace: jobSpec.Namespace(),
 							Name:      fmt.Sprintf("%s:%s", PipelineImageStream, fromTag),
 						},
 						ForcePull: true,
 						NoCache:   true,
+						Env: []coreapi.EnvVar{
+							{Name: "JOB_SPEC", Value: jobSpec.rawSpec},
```

Comment:
Should not need this, instead provide `$CLONEREFS_OPTIONS` as JSON

Reply:
So I should change this?

Reply:
I can change as well